src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3657,6 +3659,13 @@ void ApplicationManagerImpl::SubscribeAppForWayPoints(
   LOG4CXX_DEBUG(logger_,
                 "There are applications subscribed: "
                     << subscribed_way_points_apps_list_.size());
+  if (waypoints_data_) {
+    smart_objects::SmartObjectSPtr way_point_notification_ =
```

Comment:
@ZhdanovP Keep consistency in naming. Somewhere waypoint, afterwards - way_point

Reply:
@AByzhynar fixed, please take a look 3e0376f3070447c36f02454725c9403cf1e96f5b